<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PeopleMgr</class>
 <widget class="QDialog" name="PeopleMgr">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QCheckBox" name="check_11">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>120</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="check_12">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>120</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="check_13">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>120</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="del_1">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>120</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>60</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>修改数据</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>60</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>查看数据</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>60</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>201</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>用户名</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="user_1">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>120</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="del_2">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>200</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="check_23">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>210</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="user_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>210</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="check_21">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>210</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="check_22">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>210</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="del_3">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>290</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QLabel" name="user_3">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>290</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>20</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>权限管理</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="check_41">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>370</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="check_42">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>370</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="check_43">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>370</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="user_4">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>370</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="del_4">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>370</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>90</y>
     <width>821</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>50</y>
     <width>341</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>420</y>
     <width>821</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="save">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>470</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pgup">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>470</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>上一页</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pgdn">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>470</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>下一页</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="check_33">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>290</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="check_32">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>290</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="check_31">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>290</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>check_11</zorder>
  <zorder>check_12</zorder>
  <zorder>check_13</zorder>
  <zorder>del_1</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>label_6</zorder>
  <zorder>user_1</zorder>
  <zorder>del_2</zorder>
  <zorder>check_23</zorder>
  <zorder>user_2</zorder>
  <zorder>check_21</zorder>
  <zorder>check_22</zorder>
  <zorder>del_3</zorder>
  <zorder>user_3</zorder>
  <zorder>check_31</zorder>
  <zorder>check_32</zorder>
  <zorder>check_33</zorder>
  <zorder>label_5</zorder>
  <zorder>check_41</zorder>
  <zorder>check_42</zorder>
  <zorder>check_43</zorder>
  <zorder>user_4</zorder>
  <zorder>del_4</zorder>
  <zorder>line</zorder>
  <zorder>line_2</zorder>
  <zorder>line_3</zorder>
  <zorder>save</zorder>
  <zorder>pgup</zorder>
  <zorder>pgdn</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
